app/javascript/app/components/table/table.js

```diff
@@ -50,6 +57,9 @@ class TableContainer extends PureComponent {
   };
 
   setColumnWidth = column => {
+    if (this.props.hasStrechedColumns) {
```

Comment:
What do you think about a forcedColumnsWidth prop with a width like 100 instead of the hasStretchedColumns boolean. It will be more flexible